config/production.json

```diff
@@ -0,0 +1,49 @@
+{
+    "useCSRFProtection": true,
+    "security": {
+      "referrerPolicy": "origin"
+    },
+    "featureDir": [
+      {"path": "/routes/**/*.+(ts|js)"},
+      {"path": "/features/rfi/path.ts"},
+      {"path": "/features/auth/path.ts"},
+      {"path": "/features/dashboard/path.ts"},
+      {"path": "/errors/path.ts"}
+    ],
+    "PORT": "3000",
+    "AgreementService": {
+      "BASEURL": "https://dev-ccs-scale-shared-agreements-service.london.cloudapps.digital"
+    },
+    "Session": {
+      "time": "15"
+    },
+    "contentService": {
+      "BASEURL":"https://dev-ccs-scale-cat-buyer-ui.london.cloudapps.digital/",
+      "menuDirectory": "src/main/resources/content/menu-content/content.json"
+    },
+    "authenticationService": {
+      "token-endpoint": "/security/token",
+      "access_granttype": "authorization_code",
+      "token_validation_endpoint": "/security/tokens/validation",
+      "logout": "/security/log-out",
+      "logout_callback": "/logout"
+    },
+    "logger":{
+      "baseURL": "https://api.logit.io/v2"
+    },
+    "dynamic_framework": {
+      "BASEURL": "https://dev-ccs-scale-cat-buyer-ui.london.cloudapps.digital/api/template"
```

Comment:
Why we are hard codding the dev RL in production config & why we need this config file ?